packages/lokalise-download/action.yml

```diff
@@ -0,0 +1,86 @@
+name: 'Lokalise Download'
+description: 'Download files from lokalise'
+branding:
+  icon: 'flag'
+  color: 'green'
+inputs:
+  lokalise-token: # id of input
+    description: 'Your CLI/API token for Lokalise'
+    required: true
+  lokalise-project-id: # id of input
+    description: 'Your project ID for Lokalise'
+    required: true
+  file-path: # id of input
+    description: 'The path to the destination folder'
+    required: true
+  format: # id of input
+    description: 'The format of the file you want to download'
+    required: true
+    default: 'json'
+  bundle-structure: # id of input
+    description: 'The bundle structure'
+    required: true
+    default: 'src/locales/%LANG_ISO%'
+  placeholder-format:
+    description: 'Placeholder format of strings'
+    required: true
+    default: 'i18n'
+  trigger:
+    description: 'Trigger integration export'
+    required: true
+    default: 'github'
+  add_newline_eof:
+    description: 'Enable to add new line at end of file'
+    required: true
+    default: 'true'
+  filter_repositories:
+    description: 'Pull requests will be created only for listed repositories'
+    required: true
+  filter_filenames:
+    description: 'Only keys attributed to selected files will be included'
+    required: true
+  plural_format:
+    description: 'Override the default plural format for the file type'
+    required: true
+    default: 'icu'
+  indentation:
+    description: 'Provide to override default indentation in supported files'
+    required: true
+    default: '2sp'
+  export_sort:
+    description: 'Allowed value are first_added, last_added, last_updated, a_z, z_a'
+    required: true
```

Comment:
does it make sense to make it required? can we just put it as `a_z` by default?
the same of indendation, plural_format, eof, trigger, bundle structure etc

Reply:
added default values